<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddTableWidget</class>
 <widget class="QWidget" name="AddTableWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AddTableWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QFrame" name="tableNameQFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="tableNameQLabel">
        <property name="text">
         <string>nazwa tabeli</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="tableNameQLineEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QFrame" name="columnQFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0" columnstretch="1,1">
      <item row="0" column="0">
       <widget class="QLabel" name="columnNameQLabel">
        <property name="text">
         <string>nazwa kolumny</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="columnTypeQLabel">
        <property name="text">
         <string>typ danych</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="columnNameQLineEdit"/>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="columnTypeQComboBox"/>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="addColumnQPushButton">
        <property name="text">
         <string>Dodaj kolumnę</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="deleteColumnQPushButton">
        <property name="text">
         <string>Usuń kolumnę</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>236</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="saveQPushButton">
     <property name="text">
      <string>Zapisz</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="cancelQPushButton">
     <property name="text">
      <string>Anuluj</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QTableWidget" name="tableQTableWidget"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
